Code/Mantid/Framework/DataObjects/src/ReflectometryTransform.cpp

```diff
@@ -304,6 +331,18 @@ MatrixWorkspace_sptr ReflectometryTransform::executeNormPoly(
             outWS->getSpectrum(qIndex - 1)->getSpectrumNo());
         detIDMapping.push_back(detector->getID());
       }
+
+      // Debugging
+      if (vertexes) {
```

Comment:
Fine. Personally, I would have made the row creating happen inside the function, but that doesn't matter too much.